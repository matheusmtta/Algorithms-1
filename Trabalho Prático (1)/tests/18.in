14 i
39 i
142 p
142 p
31 i
48 p
3 i
154 p
39 r
3 r
9 i
7 i
19 p
31 r
25 i
9 r
25 r
112 p
168 p
29 i
7 r
17 i
14 r
38 i
145 p
50 p
36 i
37 i
17 r
29 r
147 p
20 i
37 r
152 p
20 r
15 i
40 p
32 p
57 p
