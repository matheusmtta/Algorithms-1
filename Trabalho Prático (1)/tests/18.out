5
5
3
5
3
8
12
5
3
4
4
3
4
3
